etl/vt.py

```diff
@@ -6,10 +6,11 @@
 import urllib.request
 import urllib.parse
 from collections import Counter
-from etl.utils.mongo import MongoDatabase
 import pymongo
+from nameko.rpc import rpc
+from etl.utils.mongo import MongoDatabase
 from skills_utils.iteration import Batch
-from etl.preprocessor import SingleRankWithContext
+from etl.preprocessor import SingleRankWithContext # shoudl SGRank be used, might be better?
```

Comment:
We will probably do a lot of iteration on preprocessors, I wouldn't worry about it here